docs/rules.md

```diff
@@ -60,6 +61,14 @@ You can configure options for each rule. Add an `options` key to your rule defin
 * If `false`, the rule note will not be included in the output message
 * If `not set`, `include_note` in your `woke` config file (ie `.woke.yml`) regulates if the note should be included in the output message (default: `false`).
 
+### `regex_terms`
+
+:octicons-milestone-24: Default: `false`
+
+* If `true`, terms will be evaluated as regular expressions
+* If `false`, terms will be treated as plain-text values
+* **NOTE** this is an advanced feature. Rules will be skipped if they do not compile. Only use non-capturing groups in patterns. Look-around assertions are not supported.
```

Comment:
I think a fatal error should happen if a supplied regex can't compile. Otherwise the user may never know that their rule isn't being used.

Reply:
Sure - I added a Error log message, so when it runs if a rule won't compile, an error is generated but it continues, it looks like this:

```
2021-08-13T09:03:27-05:00 ERR Unable to compile regular expression, disabling rule error="error parsing regexp: missing closing ): `(?i)(black(\\s*list)`" Rule=rulename
```
I thought about letting it do a fatal error instead, though I chose this approach instead in anticipation of the remote ruleset feature - my thought was that if there is a rule that doesn't work in a remote ruleset that the user can't necessarily directly change, it could continue on after printing the error. That said, I'm totally happy to go either way with this - if you want it a fatal error that's easy to change, I can just change it back to a MustCompile instead, or doing a panic after logging the error so that they still get that debug info from that log. What do you prefer here?

Reply:
> my thought was that if there is a rule that doesn't work in a remote ruleset that the user can't necessarily directly change

I think we should tackle this when it becomes a problem or concern.